cdap-docs/included-applications/source/etl/plugins/transforms/script.rst

```diff
@@ -43,6 +43,10 @@ will scale the ``'count'`` field by 1024.
 **schema:** The schema of output objects. If no schema is given, it is assumed that the output
 schema is the same as the input schema.
 
+**lookup:** The configuration for the lookup tables to use. For example, if lookup table "purchases" is configured,
+then you will be able to perform operations on that lookup table in your script:
```

Comment:
Right, it's any number. I agree with using "with" instead of "on", but not sure about the first sentence: "The configuration of any lookup tables being used." I'd prefer "The configuration of the lookup tables to use" because configuring the tables here allows you to use them in your script.